spec/integ/matrix-client-crypto.spec.js

```diff
@@ -542,19 +542,18 @@ describe("MatrixClient crypto", function() {
             .nodeify(done);
     });
 
-    it("Bob receives a message", function(done) {
-        Promise.resolve()
+    it("Bob receives a message", function() {
```

Comment:
Removing `done` makes the test synchronous, at least in Jasmine it does. Can you link to me the docs which show that returning a promise from a test is sufficient to make the test async?